acceptance/bundle/resources/clusters/deploy/update-and-resize-autoscale/script

```diff
@@ -0,0 +1,58 @@
+envsubst < databricks.yml.tmpl > databricks.yml
+
+cleanup() {
+    trace $CLI bundle destroy --auto-approve
+    rm out.requests.txt
+}
+trap cleanup EXIT
+
+trace $CLI bundle deploy
+
+title "Cluster should exist with num_workers after bundle deployment:\n"
+CLUSTER_ID=$($CLI bundle summary -o json | jq -r '.resources.clusters.test_cluster.id')
```

Comment:
You can do this:

`echo "$CLUSTER_ID:CLUSTER_ID" >> ACC_REPLS`

this will add to replacements and then you don't need regex in `[[Repls]]`section (which does not tell if id stays the same or is changed).